### DISCUSSION TITLE
Opening a transaction via description throws an unrecoverable error

### DISCUSSION THREAD

**Discussion by magnum61** [firefly-iii/firefly-iii#6349] [General]
Hi all,

i have Firefly installed fresh on a Raspberry 4 for testing. Will be moved later to a better server.
So far everything is working, but i am not able to view a transaction by simply clicking on the description link. 
Whenever i try that, Firefly throws an unrecoverable error on the screen.

Going back and using the "edit" function to open the transaction works. 
Any ideas how to get this fixed?

The logs as suggested by the error do not give any explanation what happened.

Only on the error page itself this is shown:

`FireflyIII\Transformers\TransactionGroupTransformer::getAmount(): Argument #2 ($amount) must be of type string, float given, called in /var/www/html/public/app/Transformers/TransactionGroupTransformer.php on line 355`

Thanks in advance

**Comment by JC5** [comment]
This is an issue specific to SQLite, which will be fixed in the next minor update.

**Reply by magnum61** [reply]
Got it, thanks

This is my testing site to check if Firefly is the right one for me. 
If i go to production i will eiter use MySQL or Postgres

**Reply by JC5** [reply]
Then you won't run into trouble 👍🏻

**Comment by magnum61** [comment]
Quick Update:

I installed Firefly now on a virtual machine running Ubuntu Server with Apache and MariaDB
All issues are gone, works without any trouble.

Thanks, i really like your work. I was looking pretty long for a financial manager that fits to my needs.
